src/unity/toolkits/activity_classification/activity_classifier.cpp

```diff
@@ -65,33 +65,58 @@ std::vector<std::string> get_supported_metrics() {
           "confusion_matrix", "roc_curve"};
 }
 
-std::tuple<float,size_t> get_accuracy(size_t prediction_window, size_t num_classes, shared_float_array output_chunk, shared_float_array classes_chunk, data_iterator::batch::chunk_info info ){
+std::tuple<float,size_t> get_accuracy_per_sequence(size_t prediction_window, size_t num_classes, 
```

Comment:
Sorry, as we iterate on this, I keep seeing new ways to simplify things. And I think we should err on the side of simplification!

Thinking more about this, I don't think we need to return a tuple. The `num_predictions` value we're returning can actually be computer (by the caller) relatively simply. (I think what I was doing in the prediction code was probably overkill.) Which means that this function can boil down just to counting the number of correct predictions in one sequence ("batch row").

I suspect you'll find that we can reduce this to e.g.
```
size_t count_correct_predictions(size_t num_classes, size_t num_predictions, const shared_float_array& output_chunk, const shared_float_array& label_chunk) {
```